docs/en/remote-flow.rst

```diff
@@ -327,8 +329,7 @@ The JWS payload parameters are described herein:
 
   - ``presentation_definition``: JSON object according to `Presentation Exchange <https://identity.foundation/presentation-exchange/spec/v2.0.0/>`_. This parameter MUST not be present when ``presentation_definition_uri`` or ``scope`` are present.
   - ``presentation_definition_uri``: Not supported. String containing an HTTPS URL pointing to a resource where a Presentation Definition JSON object can be retrieved. This parameter MUST be present when ``presentation_definition`` parameter or a ``scope`` value representing a Presentation Definition is not present. 
-  - ``client_metadata``: A JSON object containing the Relying Party metadata values. The ``client_metadata`` parameter MUST NOT be present when ``client_id_scheme`` is ``entity_id``. Since the ``client_metadata`` is taken from ``trust_chain``, this parameter is intended to not be used.
-  - ``client_metadata_uri``: string containing an HTTPS URL pointing to a resource where a JSON object with the Relying Party metadata can be retrieved. The ``client_metadata_uri`` parameter MUST NOT be present when ``client_id_scheme`` is ``entity_id``. Since the ``client_metadata`` is taken from ``trust_chain``, this parameter is intended to not be used.
+  - ``client_metadata``: A JSON object containing the Relying Party metadata values. If the ``client_metadata`` parameter is present when ``client_id_scheme`` is ``entity_id``, the client_metadata MUST only contains the member ``jwks`` and containing within it ephemeral cryptographic public keys to be used only for the Wallet Instance with which the Relying Party is interacting with.
```

Comment:
not sure to want to keep this.

even if there are claims about the importance of using ephemeral keys by RP, there might be other concerns about using jwks outside of the trust chain, that would allow an adversary to hijack the response and therefore be able to decrypt it.

further conversation are required, even within the DCP WG  and with the italian team as well.

Reply:
To me, the purpose of this change is slightly unclear. 
Does it imply that `client_metadata` in Request object goes from "MUST NOT be present" to "OPTIONAL: when used it must contain an ephemeral JWKS"?
If so, where did the necessity of ephemeral JWKS arise from?

Reply:
from Google Credential API, something that will be formally included later on

at the current stage I'd prefer and suggest to move this out from this PR and handle it in a separate issue

from my perspective even using credential api it is still unclear this requirement.

Reply:
Ah, I see
I'll try to learn more about Google Credential API in the near future as I don't know anything about it.

Reply:
it is now included within the openid4vp specs, annex, in the editor's copy on github

Reply:
@Zicchio I have moved it out, to this issue: https://github.com/italia/eudi-wallet-it-docs/issues/408